xcode/Subconscious/Shared/UIKit/BlockEditor/Blocks/ListBlockCell.swift

```diff
@@ -0,0 +1,311 @@
+            textView.translatesAutoresizingMaskIntoConstraints = false
+            textView.delegate = self
+            textView.inputAccessoryView = toolbar
+            listContainer.addSubview(textView)
+            
+            listContainer.addSubview(bulletView)
+            
+            transcludeListView.directionalLayoutMargins = transcludeMargins
+            stackView.addArrangedSubview(transcludeListView)
+
+            selectView.translatesAutoresizingMaskIntoConstraints = false
+            contentView.addSubview(selectView)
+
+            let listContainerGuide = listContainer.layoutMarginsGuide
+            NSLayoutConstraint.activate([
+                selectView.leadingAnchor.constraint(
+                    equalTo: contentView.leadingAnchor,
+                    constant: AppTheme.unit
+                ),
+                selectView.trailingAnchor.constraint(
+                    equalTo: contentView.trailingAnchor,
+                    constant: -1 * AppTheme.unit
+                ),
+                selectView.topAnchor.constraint(
+                    equalTo: contentView.topAnchor
+                ),
+                selectView.bottomAnchor.constraint(
+                    equalTo: contentView.bottomAnchor
+                ),
+                bulletView.leadingAnchor.constraint(
+                    equalTo: listContainer.leadingAnchor,
+                    constant: AppTheme.unit4
+                ),
+                bulletView.topAnchor.constraint(
+                    equalTo: listContainer.topAnchor,
+                    constant: AppTheme.unit2
+                ),
+                textView.leadingAnchor.constraint(
+                    equalTo: listContainerGuide.leadingAnchor
+                ),
+                textView.trailingAnchor.constraint(
+                    equalTo: listContainerGuide.trailingAnchor
+                ),
+                textView.topAnchor.constraint(
+                    equalTo: listContainerGuide.topAnchor
+                ),
+                textView.bottomAnchor.constraint(
+                    equalTo: listContainerGuide.bottomAnchor
+                ),
+                stackView.leadingAnchor.constraint(
+                    equalTo: leadingAnchor
+                ),
+                stackView.trailingAnchor.constraint(
+                    equalTo: trailingAnchor
+                ),
+                stackView.topAnchor.constraint(
+                    equalTo: topAnchor
+                ),
+                stackView.bottomAnchor.constraint(
+                    equalTo: bottomAnchor
+                )
+
+            ])
+        }
+        
+        required init?(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+        }
+        
+        private func createBulletView() -> UIView {
+            let bulletSize: CGFloat = 6
+            let frameWidth: CGFloat = 8
+            let lineHeight: CGFloat = 22
```

Comment:
Do we need to consider system preferences for font size etc?

Reply:
Eventually, yeah. For now I would like to leave this as-is, since I'm mostly blocking out the UI and have yet to get to details like font-size adjustment.